src/data/actions/courseInfo.js

```diff
@@ -66,15 +81,52 @@ export function fetchCourseInfo(id) {
   };
 }
 
-export function createCourse(courseData) {
+export function createCourseRun(course, courseRunData) {
+  return (dispatch) => {
+    dispatch(createNewCourseRun());
+    if (!course.key) {
+      dispatch(courseCreateRunFail('Course Run create failed, please try again or contact support. Error( Course create incomplete )'));
+      return null;
+    }
+
+    const data = {
+      course: course.key,
+      start: courseRunData.start,
+      end: courseRunData.end,
+    };
+    return DiscoveryDataApiService.createCourseRun(data)
+      .then((response) => {
+        const courseRun = response.data;
+        dispatch(courseCreateRunSuccess(courseRun));
+        return dispatch(push(`/courses/${course.uuid}/`));
+      })
+      .catch((response) => {
+        let errorMsg = `Course Run create failed, please try again or contact support. \nError ( ${response.message} )`;
+        const errorData = response.response.data;
```

Comment:
Should this be response.data? Just want to double check that response.response is desired.

Reply:
This is the way the data was coming from the server.  Are we doing something incorrectly when we send the response from the server side?

Reply:
Maybe? But I don't think I did anything special. Just gave DRF the exception.